src/components/LessonListsByStates/index.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react'
+import {
+  newLessonsActionText,
+  inProgressTitleText,
+  inQueueTitleText,
+  publishedTitleText,
+  noInProgressLessonsDescriptionText,
+  noInQueueLessonsDescriptionText,
+  noPublishedLessonsDescriptionText,
+} from 'utils/text'
+import {inProgressLessonStates, publishedLessonStates} from 'utils/lessonStates'
+import LessonList from 'components/LessonList'
+import Tabs from 'components/Tabs'
+import Prompt from 'components/Prompt'
+
+export default ({instructor}) => (
+  <Tabs groups={[
+    {
+      title: inProgressTitleText,
+      component: (
+        <LessonList
+          states={inProgressLessonStates}
+          fallback={
+            <div className='pa3'>
+              <Prompt
+                description={noInProgressLessonsDescriptionText}
+                actionText={newLessonsActionText}
+                action={'/lessons/new'}
+              />
+            </div>
+          }
+          instructor={instructor}
+        />
+      ),
+    },
+    {
+      title: inQueueTitleText,
+      component: (
+        <LessonList
+          states={['approved']}
+          fallback={
+            <div className='pa3'>
+              <Prompt
+                description={noInQueueLessonsDescriptionText}
+                actionText={newLessonsActionText}
+                action={'/lessons/new'}
+              />
+            </div>
+          }
+          instructor={instructor}
+        />
+      ),
+    },
+    {
+      title: publishedTitleText,
+      component: (
+        <LessonList
+          states={publishedLessonStates}
+          fallback={
+            <div className='pa3'>
+              <Prompt
+                description={noPublishedLessonsDescriptionText}
+                actionText={newLessonsActionText}
+                action={'/lessons/new'}
+              />
+            </div>
+          }
+          instructor={instructor}
+        />
+      ),
+    },
+  ]} />
+)
```

Comment:
@joelhooks I agree the buttons seem a bit big, so what I mean is the change should be made from the style guide down instead of using `small` everywhere